akka-contrib/src/multi-jvm/scala/akka/contrib/pattern/DistributedPubSubMediatorSpec.scala

```diff
@@ -471,6 +471,49 @@ class DistributedPubSubMediatorSpec extends MultiNodeSpec(DistributedPubSubMedia
       }
       enterBarrier("after-14")
     }
+
+    "get topics after simple publish" in within(15 seconds) {
+      runOn(first) {
+        val s1 = Subscribe("topic_a1", createChatUser("u14"))
+        mediator ! s1
+
+        expectMsg(SubscribeAck(s1))
+
+        val s2 = Subscribe("topic_a1", createChatUser("u15"))
+        mediator ! s2
+        expectMsg(SubscribeAck(s2))
+
+        val s3 = Subscribe("topic_a2", createChatUser("u16"))
+        mediator ! s3
+        expectMsg(SubscribeAck(s3))
+      }
+
+      runOn(second) {
+        val s3 = Subscribe("topic_a1", createChatUser("u17"))
+        mediator ! s3
+        expectMsg(SubscribeAck(s3))
+      }
+
+      enterBarrier("topics-registered")
+
+      runOn(first) {
+        mediator ! GetTopics
+        expectMsgPF() {
+          case CurrentTopics(topics) if topics.contains("topic_a1")
+            && topics.contains("topic_a2") ⇒ true
+        }
+      }
+
+      runOn(second) {
+        mediator ! GetTopics
+        expectMsgPF() {
+          case CurrentTopics(topics) if topics.contains("topic_a1") ⇒ true
+        }
+      }
```

Comment:
we should also verify that topic_a2 is eventually replicated, something like this

      runOn(second) {
        // topics will eventually be replicated
        awaitAssert {
          mediator ! GetTopics
          val topics = expectMsgType[CurrentTopics].topics.keys.toSet
          topics should contain("topic_a1")
          topics should contain("topic_a2")
        }
      }